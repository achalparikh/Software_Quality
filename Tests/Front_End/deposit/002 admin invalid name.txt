login
admin
deposit 
CLARK KENT
logout
quit