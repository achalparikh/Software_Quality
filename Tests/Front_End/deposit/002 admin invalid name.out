Enter user type - "standard" or "admin":
You are now logged in as an admin
COMMAND: deposit - enter user:
ERROR: User "CLARK KENT" does not exist. Try again
You are now logged out
